Include plugin overrides.zcml before plugin.zcml.
Solves a conflict in philippines that includes overrides.zcml itself.
Index: schooltool/src/schooltool/skin/flourish/instance/configure.zcml
===================================================================
--- schooltool.orig/src/schooltool/skin/flourish/instance/configure.zcml
+++ schooltool/src/schooltool/skin/flourish/instance/configure.zcml
@@ -54,7 +54,7 @@
   <include package="schooltool.level.browser" file="flourish.zcml" />
   <include package="schooltool.export" file="flourish.zcml" />
 
-  <includePlugins package="schooltool" file="plugin.zcml" />
   <includePluginsOverrides package="schooltool" />
+  <includePlugins package="schooltool" file="plugin.zcml" />
 
 </configure>
